template.json

```diff
@@ -27,6 +27,7 @@
       "react-scripts": "4.0.3"
     },
     "devDependencies": {
+      "@hipo/react-ui-toolkit": "^1.0.0-alpha.5.1.1",
```

Comment:
I think this should be normal dependency as it will be included in the final bundle